src/rest.cpp

```diff
@@ -265,6 +267,112 @@ static bool rest_block_notxdetails(HTTPRequest* req, const std::string& strURIPa
     return rest_block(req, strURIPart, false);
 }
 
+static bool rest_block_filter(HTTPRequest* req, const std::string& strURIPart)
+{
+    if (!CheckWarmup(req))
+        return false;
+    std::string param;
+    const RetFormat rf = ParseDataFormat(param, strURIPart);
+
+    //request is sent over URI scheme /rest/blockfilter/filtertype/requesttype/blockhash
+    std::vector<std::string> uriParts;
+    boost::split(uriParts, param, boost::is_any_of("/"));
+    if (uriParts.size() != 3) {
+        return RESTERR(req, HTTP_BAD_REQUEST, "Invalid URI format. Expected /rest/blockfilter/filtertype/requesttype/blockhash");
+    }
+
+    uint256 block_hash;
+    if (!ParseHashStr(uriParts[2], block_hash)) {
```

Comment:
Nit, the use of a single line if statement would be better

Reply:
Would be much too long a line to do that.

Reply:
Oh sorry I meant to ignore the brackets...